.eslintrc

```diff
@@ -1,10 +1,43 @@
 extends: 'airbnb-base'
 env:
-  node: false
-  browser: true
-  commonjs: true
+  node: true
+  browser: false
 rules:
 
+  # Forces alignment of objects values and colons of both single line objects
+  # and multiLine objects. This promotes smaller diffs.
+  key-spacing:
+    - 2
+    - singleLine:
+        beforeColon: false
+        afterColon: true
+      multiLine:
+        beforeColon: true
+        afterColon: true
+        align: colon
```

Comment:
That's a good point. So following this argument, no alignment should (AT ALL) should be on the rules.